src/main/scala/nl/vroste/zio/kinesis/client/DefaultClientsLayers.scala

```diff
@@ -0,0 +1,31 @@
+package nl.vroste.zio.kinesis.client
+
+import software.amazon.awssdk.services.cloudwatch.CloudWatchAsyncClient
+import software.amazon.awssdk.services.dynamodb.DynamoDbAsyncClient
+import software.amazon.awssdk.services.kinesis.KinesisAsyncClient
+import zio.{ Has, ZIO, ZLayer, ZManaged }
+
+object DefaultClientsLayers {
+
+  def kinesisAsyncClientLayer: ZLayer[Any, Throwable, Has[KinesisAsyncClient]] =
```

Comment:
Maybe we should provide methods that allow access to the builder for customization.

Reply:
I will have a go

Reply:
done